pocs/contract-example/vesting-flow.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Vesting Flow</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="/dist/local-importmap.js"></script>
+  </head>
+  <body>
+    <div>
+      <h1>Contract Examples > Vesting</h1>
+      <h2>1. Setup Environment</h2>
+
+      <label for="runtimeUrl">URL to a Marlowe Runtime instance:</label>
+      <input
+        id="runtimeUrl"
+        type="text"
+        autocomplete="on"
+        placeholder="http://localhost:32952"
+      />
+      <br />
+      <label for="wallet">Select wallet:</label>
+      <select id="wallet" name="wallet">
+        <option value="nami">Nami</option>
+        <option value="eternl">Eternl</option>
+      </select>
+      <div>
+        <h3>1.1 Create Vesting contract</h3>
+        <label for="contract">JSON Contract:</label>
+        <textarea
+          id="contract"
+          type="text"
+          style="width: 100%; height: 20em"
+        ></textarea>
+      </div>
+      <br />
+      <input id="generate-contract" type="button" value="Generate Contract" />
+      <input
+        id="create-contract"
+        type="button"
+        value="Generate | Create Marlowe Contract on Cardano"
+      />
+
+      <div>
+        <h3>1.2 Apply Inputs</h3>
+        <label for="contract-id">contract Id : </label>
+        <input id="contract-id" type="text" autocomplete="on" />
+        <input id="apply-input" type="button" value="Apply Input" />
+        <h4>Contracts waiting for an action</h4>
+        <div id="contracts-status"></div>
+      </div>
+      <hr />
+      <h2>Console</h2>
+      <div id="console"></div>
+    </div>
+  </body>
+  <script type="module">
+    import { clearConsole, log } from "../js/poc-helpers.js";
+    import { mkRuntimeLifecycle } from "@marlowe.io/runtime-lifecycle/browser";
+    import { Vesting } from "@marlowe.io/language-examples";
+    import { MarloweJSON } from "@marlowe.io/adapter/codec";
+    import { mkRestClient } from "@marlowe.io/runtime-rest-client";
+    import { Deposit, Choice } from "@marlowe.io/language-core-v1/next";
+    const dappId = "marlowe.ts.sdk.example.vesting.v1.0.4";
+    const runtimeURL =
```

Comment:
Note that I added `getRuntimeUrl` and `setupLocalStorageRuntimeUrl` in the `poc-helpers` module. You can change the default from `"http://localhost:32952"` to `"https://marlowe-runtime-preprod-web.scdev.aws.iohkdev.io/"` but it is helpful to use the same helper